docs/whats-new/2020-12.md

```diff
@@ -0,0 +1,123 @@
+---
+title: ".NET docs: What's new for December 2020"
+description: "What's new in the .NET docs for December 2020."
+ms.date: 01/04/2021
+---
+
+# .NET docs: What's new for December 2020
+
+Welcome to what's new in the .NET docs for December 2020. This article lists some of the major changes to docs during this period.
+
+## .NET
+
+### New articles
+
+- [BinaryFormatter event source](../standard/serialization/binaryformatter-event-source.md) - BinaryFormatter event source doc
+
+### Updated articles
+
+- [Resolve assembly loads](../standard/assembly/resolve-loads.md) - Assembly loading
+
+## .NET Core
+
+### New articles
+
+- [Debugging StackOverflow errors](../core/diagnostics/debug-stackoverflow.md) - Add debug-stackoverflow.md
+- [SOS debugging extension](../core/diagnostics/sos-debugging-extension.md) - Documentation for SOS
+- [Well-known event providers in .NET](../core/diagnostics/well-known-event-providers.md) - Add docs on well-known event providers
+- [Blazor: Route precedence logic changed in Blazor apps](../core/compatibility/aspnet-core/5.0/blazor-routing-logic-changed.md) - Document Blazor route matching precedence logic
```

Comment:
We should find a way to route ASP.NET Core breaking changes to the ASP.NET Core what's new in docs article.

Reply:
Adding @scottaddie 

For the moment, we should just copy & paste, I think.